crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java

```diff
@@ -51,16 +51,23 @@ public void testGenerateBip39Wallets() throws Exception {
     @Test
     public void testGenerateFullNewWalletFile() throws Exception {
         String fileName = WalletUtils.generateFullNewWalletFile(PASSWORD, tempDir);
-        testGenerateNewWalletFile(fileName);
+        testGeneratedNewWalletFile(fileName);
     }
 
+    @Test
+    public void testGenerateNewWalletFile() throws Exception {
+        String fileName = WalletUtils.generateLightNewWalletFile(PASSWORD, tempDir);
```

Comment:
We should default to the full wallet file - see https://github.com/ethereum/go-ethereum/blob/8f8774cf6dd5bbcba9fa3773fa34d13d6523a147/node/config.go#L376

Reply:
Well, it has been changed default to the full wallet file.

Reply:
Well, it has been changed as the full wallet file by default.